src/console.cpp

```diff
@@ -274,7 +228,7 @@ static void IConsoleAliasExec(const IConsoleAlias *alias, byte tokencount, char
 	DEBUG(console, 6, "Requested command is an alias; parsing...");
 
 	if (recurse_count > ICON_MAX_RECURSE) {
-		IConsoleError("Too many alias expansions, recursion limit reached. Aborting");
+		IConsolePrint(CC_ERROR, "Too many alias expansions, recursion limit reached. Aborting");
```

Comment:
I guess we can remove all these `aborting` statements now, as it is much more clear that is happening?